x-pack/plugins/observability_solution/apm/public/components/app/entities/logs/logs_service_overview.tsx

```diff
@@ -52,8 +85,60 @@ export function LogsServiceOverview() {
       end={end}
     >
       <ChartPointerEventContextProvider>
-        <AddAPMCallOut />
-        <EuiSpacer size="l" />
+        {isLogsApmCalloutEnabled ? (
+          <>
+            <AddAPMCallOut
+              onClose={() => {
+                setIsLogsApmCalloutEnabled(false);
+              }}
+            />
+            <EuiSpacer size="l" />
+          </>
+        ) : null}
+        {data?.entity.hasLogMetrics === false ? (
```

Comment:
this one fails when you select a timerange where latest index doesn't have the service. For example, last 4 minutes

Reply:
yeah, I think we can live with that.